src/aioauth/requests.py

```diff
@@ -2,13 +2,13 @@
 
 from .config import Settings
 from .structures import CaseInsensitiveDict
-from .types import CodeChallengeMethod, GrantType, RequestMethod, ResponseType
+from .types import CodeChallengeMethod, GrantType, RequestMethod
 
 
 class Query(NamedTuple):
     client_id: Optional[str] = None
     redirect_uri: str = ""
-    response_type: Optional[ResponseType] = None
```

Comment:
Wouldn't this default to `ResponseType.TYPE_NONE`?

Reply:
this class is general. when it's grant type request, we don't need response_type in Query. that's why it is optional here